source/05-rspec/02-matchers.md

```diff
@@ -42,23 +42,23 @@ For example:
 
 ```ruby
 expect(10).to be > 5
-expect([1, 2, 3]).to (1)
+expect([1, 2, 3]).to include 1
 expect("Ruby Monstas").to start_with "Ruby"
 ```
 
-No matter how exactly the code works that implements these methods `expect`, `to`,
-and, for example, `eq` or `start_with`: The purpose is being able to formulate
-code that kinda reads like a sentence. You'll get used to these pretty soon, once
+No matter exactly how the code that implements these methods `expect`, `to`,
+and, for example, `eq` or `start_with` works: the purpose is to allow for
+code that kinda reads like an English sentence. You'll get used to these pretty soon, once
 you've started writing some RSpec tests.
 
 Essentially, you start with `expect(whatever_thing_to_test).to`, and then you find
-a matcher that works. `eq` always is a good start. So you end up with:
+a matcher that works. `eq` is always a good start. So you end up with:
 
 ```ruby
 expect(whatever_thing_to_test).to eq whatever_you_expect
 ```
 
-E.g.:
+For example:
```

Comment:
I noticed you've used "e.g." and "i.e." a lot, and I think, in some cases, it doesn't look or read so well.